drivers/age-swift/Tests/age-swiftTests/age_swiftTests.swift

```diff
@@ -0,0 +1,24 @@
+import XCTest
+@testable import age_swift
+
+let host = "127.0.0.1";
+let port = 5432 ;
+let user = "root";
+let db_name = "demo";
+
+final class age_swiftTests: XCTestCase {
+    func testExample() throws {
```

Comment:
Suggest to name the test method as following: 

```scenarioName_should_expectedOutcome()```

For example: 

```connectingWithSuppliedParams_shouldNotError()``` 

Suggest to also add some more scenarios: 

* What happens when the incorrect params are supplied? What's the expected error message? Ensure it does not crash. 
* What happens when we execute CYPHER statements returning vertices? 
* What happens when we execute CYPHER statements returning vertices and edges?

Reply:
Yes will keep adding new test cases, in my upcoming pull requests

Reply:
You can add more commits to this PR. They can be squashed down when merging.